articles/virtual-desktop/tenant-setup-azure-active-directory.md

```diff
@@ -58,7 +58,8 @@ To assign the TenantCreator application role to your global administrator accoun
 3. Select **Users and groups**, then select **Add user**.
 4. Select Users and groups in the **Add Assignment** blade
 5. Search for a user account that will create your Windows Virtual Desktop tenant. 
-   - For simplicity, this can be the global administrator account.
+   - For simplicity, this can be the global administrator account. 
+   - If you are using Microsoft Identifty Provider such as contosoadmin@live.com or contosoadmin@outlook.com, signing in to Windows Virtual Desktop might be unsuccessful. It is recommended to use domain specific account such as admin@contoso.com or admin@contoso.onmicrosoft.com instead.
```

Comment:
If you're using a Microsoft Identity Provider like contosoadmin@live.com or contosoadmin@outlook.com, you might not be able to sign in to Windows Virtual Desktop successfully. We recommend using a domain-specific account like admin@contoso.com or admin@contoso.onmicrosoft.com instead.